pkg/controller/hostpathprovisioner/contoller_test.go

```diff
@@ -144,7 +148,7 @@ var _ = Describe("Controller reconcile loop", func() {
 		sa = &corev1.ServiceAccount{}
 		err = cl.Get(context.TODO(), saNN, sa)
 		Expect(err).NotTo(HaveOccurred())
-		Expect(sa.ObjectMeta.Labels["k8s-app"]).To(Equal("test-name"))
+		Expect(sa.ObjectMeta.Labels["k8s-app"]).To(Equal("hostpath-provisioner"))
```

Comment:
Use the const you just create in common.go?

Reply:
Done